pkg/web/routes.go

```diff
@@ -121,20 +121,60 @@ func NewServer(params ServerParams) (*Server, error) {
 	return &Server{tmpl: tmpl, params: params}, nil
 }
 
+/* proxyable functions allow test of routing functionality independently */
+
+func listReposDefault(stores context2.Stores) []model.RepoDescriptor {
+	res, err := core.ListRepos(stores)
+	if err != nil {
+		panic(err)
+	}
+	return res
+}
+
+func listBundlesDefault(repoName string, stores context2.Stores) []model.BundleDescriptor {
+	res, err := core.ListBundles(repoName, stores)
+	if err != nil {
+		panic(err)
+	}
+	return res
+}
+
+func listBundleFilesDefault(repoName string, bundleID string, stores context2.Stores) []model.BundleEntry {
+	bundle := core.NewBundle(core.NewBDescriptor(),
+		core.Repo(repoName),
+		core.ContextStores(stores),
+		core.BundleID(bundleID),
+	)
+	err := core.PopulateFiles(context.Background(), bundle)
+	if err != nil {
+		panic(err)
```

Comment:
TODO (nice to have): handle error and render error w/ template

Reply:
i've spun off a task here..